server/sonar-web/src/main/js/app/components/help/GlobalHelp.js

```diff
@@ -28,7 +28,9 @@ import TutorialsHelp from './TutorialsHelp';
 import { translate } from '../../../helpers/l10n';
 
 type Props = {
+  currentUser: { isLoggedIn: boolean },
```

Comment:
Is it passed there only to re-render the component when changing ?  I can't find any use of it. Maybe add a comment.